chart/values.yaml

```
@@ -667,3 +670,18 @@ obs:
         http: 90011
         # NodePort associated with https port
         https: 90010
+
+localpv-provisioner:
+  # -- Enables the openebs dynamic-localpv provisioner. If disabled, modify etcd and loki-stack storage class accordingly.
+  enabled: true
+  # Enable/disable the openebs NDM sub-chart. It's recommended to keep this disabled.
+  openebsNDM:
+    enabled: false
+  # Enable/disable the creation of the openebs-device StorageClass. It's recommended to keep this disabled.
+  deviceClass:
+    enabled: false
+  hostpathClass:
+    # -- Name of the default hostpath StorageClass. If modified, also modify etcd and loki-stack default storage class accordingly.
+    name: "mayastor-openebs-hostpath"
+    # -- Path on the host where local volumes of this storage class are mounted under.
+    basePath: "/var/local/{{ .Release.Name }}"
```

To ensure it doesn't collide with io-engine

```suggestion
    basePath: "/var/local/{{ .Release.Name }}/localpv"
```